### PR TITLE
Norik

### PR DESCRIPTION
**Pull request by NoroAxper**

### CODE REVIEW

File: src/client/App.jsx [hunk 1]
```diff
@@ -1,11 +1,152 @@
...
+  password: ''
+}
+const moviesState = {
+  movieList: [],
+  title: '',
+  description: '',
+  runtimeMins: 0,
+}
 function App() {
+  const [newUser, setNewUser] = useState(InitialNewUser)
```
Comment (ManNavic) [inline src/client/App.jsx:20]: I think you could use one state for register and login

File: src/client/App.jsx [hunk 2]
```diff
@@ -1,11 +1,152 @@
...
+}
+const InitialUserLogIn = {
+  username: '',
+  password: ''
+}
+const moviesState = {
+  movieList: [],
+  title: '',
+  description: '',
+  runtimeMins: 0,
```
Comment (ManNavic) [inline src/client/App.jsx:17]: runtimeMins: 0 Doesn't this overwrite the placeholder in your input ?

**Review by ManNavic — COMMENTED** [review]
Well done Norik 👍

File: src/client/App.jsx [hunk 3]
```diff
@@ -1,11 +1,152 @@
...
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify(user)
+    }
+  
+    fetch(`${apiUrl}/user/register`, options)
+    .then(function (response) {
+        return response.json()
+    })
+    e.target.reset()
```
Comment (sarajbeazley) [inline src/client/App.jsx:51]: I don't know what this does

File: src/client/App.jsx [hunk 4]
```diff
@@ -1,11 +1,152 @@
...
+          "Content-Type": "application/json"
+      },
+      body: JSON.stringify(logInDetails)
+    }
+    fetch(`${apiUrl}/user/login`, options)
+    .then(function (response) {
+        return response.json()
+    })
+    .then(data => {
+      setToken(data.token)
```
Comment (sarajbeazley) [inline src/client/App.jsx:79]: is it not here that the token needs to be saved to local storage?

File: src/client/App.jsx [hunk 5]
```diff
@@ -1,11 +1,152 @@
...
+      <h2>Movie List</h2>
+      {
+        movies.movieList.map((movie, idx) => {
+          return <div key={idx}>
+            <h3>{movie.title}</h3>
+            <p>Description: {movie.description}</p>
+            <p>Runtime: {movie.runtimeMins}</p>
+          </div>
+        })
+      }
```
Comment (sarajbeazley) [inline src/client/App.jsx:149]: Great work, will you break it down into components now?

**Pull request closed without merging**